index.html

```diff
@@ -15,26 +15,27 @@
     <div class="app">
       <header>
         <h1>Take a walk</h1>
+        <p>Ever wanted to take a walk but procrastinated until it's either dark or rainy, or both? Now you can still procrastinated, while also visualize how many precious little sunny and dry moments you wasted not walking. Hopefully that will motivate you into going out every sun/dry moment can for a walk.</p>
```

Comment:
Ever wanted to take a walk but instead procrastinated until it was either dark,  rainy, or both? Now you can still procrastinate, while also visualizing how many precious sunny and dry moments you wasted _not_ walking. Hopefully, that will motivate you to take a walk when it's sunny and dry!